docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+      nearestOwnerPropTypes: {
+        add: React.PropTypes.func.isRequired
+      },
+
+      handleSubmit: function (e) {
+        e.preventDefault();
+        var todoTextInput = this.refs.textInput.getDOMNode();
+
+        this.nearestOwnerProps.add({
+          text: todoTextInput.value
+        });
+
+        todoTextInput.value = '';
+      },
+
+      render: function () {
+        return (
+          <form onSubmit={this.handleSubmit}>
+            <input type="text" ref="textInput" placeholder="Add a Todo" />
+            <button type="submit">Add</button>
+          </form>
+        );
+      }
+    });
+
+    module.exports = TodoCreateForm;
+```
+
+## todoActions.js
+`Tux` leverages the `Flux` architecture but abstracts away all of the normal boilerplate associated with `Flux`. In `Tux` we build `actionCategories` and then we can invoke them in order to dispatch their actions and register with them in order to receive these actions. Here is the `todoActions` category that we required in from before.
+
+```javascript
+    var Actions = require('tux/Actions');
+
+    var todoActions = Actions.createActionCategory({
+      category: 'todos',
+      source: 'todo views',
+      actions: ['add', 'remove'] // the specified actions will become methods on the todoActions ActionCategory. These actions allow us to get data from our views to our actions and then to our stores
+    });
+
+    module.exports = todoActions;
+```
+
+## todoStore.js
+`Tux` provides all of the glue code needed to build stores and register them with the `TuxActions` dispatcher. Now that we have created our actions and views. Let's create our store for our Todos that respond to our actions and provide the data for our views. `Tux` here again significantly reduces the boilerplate required to get a store up and running.
+
+```javascript
+    var ActionStores = require('tux/Stores/ActionStores');
+
+    var todoStore = ActionStores.createStore({
+      _todos: [],
+
+      getAll: function () {
+        return this._todos;
+      },
+
+      onAdd: function (todo) {
+        this._todos.push({
+          text: todo.text,
+          id: this._todos.length
+        });
+        this.emitChange();
+      },
+
+      onRemove: function (todo) {
+        this._todos.splice(todo.id, 1);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          todos: {
+            add: this.onAdd,
+            remove: this.onRemove
+          }
+        };
+      }
+    });
+
+    module.exports = todoStore;
+```
+
+## TuxTodoApp.js and index.html
+Finally after putting together all of the pieces, let's render the `TodoViewOwner` component to the DOM.
+
+```javascript
+    var React = require('tux/React');
+    var TodoViewOwner = require('./TodoViewOwner.jsx');
+
+    // Render the TodoViewOwner component to the element with the id 'main'
+    React.render(<TodoViewOwner />, document.getElementById('main'));
+```
+
```

Comment:
Add line: below is our index.html file. The `<div id="main"></div>` will be where our `TodoViewOwner` is rendered and the `<script src="bundle.js"></script>` will refer to the output of `browserify`.